pkg/apm/application_instances.go

```diff
@@ -0,0 +1,74 @@
+package apm
+
+import "fmt"
+
+// ApplicationInstanceLinks represents all the links for a New Relic application instance.
+type ApplicationInstanceLinks struct {
+	Application     int `json:"application,omitempty"`
+	ApplicationHost int `json:"application_host,omitempty"`
+}
+
+// ApplicationInstance represents information about a New Relic application instance.
+type ApplicationInstance struct {
+	ID              int                       `json:"id,omitempty"`
```

Comment:
Not a huge deal in all likelihood, but would we want to remove `omitempty` for an ID in case an ID is zero?

Reply:
I guess in general I'm wondering about usage of `omitempty` in general for integers than could be valid values of `0`.

Reply:
This is a read-only resource, so it's probably a non-issue?

Reply:
Si, de acuerdo